cds_ils/ldap/api.py

```diff
@@ -419,19 +419,33 @@ def import_new_ldap_users(new_ldap_users, log_func):
             # Apparently, in some cases, there could be multiple LDAP users
             # with different person id but same email.
             email = ldap_user_get_email(ldap_user)
+            employee_id = ldap_user_get(ldap_user, "employeeID")
```

Comment:
You might want, maybe, to add the extra data that they need in this PR (they requested the mail box and phone maybe?)
Or for another time ;)